bpf-notifier/notifier.c

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause)
+/* Copyright (c) 2023 Red Hat */
+
+#include <argp.h>
+#include <signal.h>
+#include <stdio.h>
+#include <time.h>
+#include <unistd.h>
+#include <sys/resource.h>
+#include <bpf/libbpf.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <net/if.h>
+#include <linux/if_link.h>
+#include <linux/if_ether.h>
+#include <bpf/bpf.h>
+#include <netinet/in.h>
+#include <netinet/ether.h>
+#include <arpa/inet.h>
+#include <string.h>
+
+#include "notifier.skel.h"
+#include "notifier.h"
+
+static struct env {
+	int interval;
+	bool verbose;
+} env = { 1, 0 };
+
+const char *argp_program_version = "notifier 0.0";
+const char *argp_program_bug_address = "<donald.hunter@redhat.com>";
+const char argp_program_doc[] =
+"BPF notifier prototype.\n"
```

Comment:
I think the indentation is a little off here...

Reply:
Yeah. Welp.